concourse/pipelines/deploy.yml

```diff
@@ -422,6 +426,9 @@ jobs:
             cd govuk-infrastructure/terraform/deployments/govuk-publishing-platform
 
             terraform init -backend-config "role_arn=$ASSUME_ROLE_ARN"
+
+            terraform workspace select "$WORKSPACE"
```

Comment:
Sorry I think we will need to do the same for the `destroy-terraform` job.

Reply:
yep